common/djangoapps/microsite_configuration/backends/database.py

```diff
@@ -126,6 +127,8 @@ class DatabaseMicrositeTemplateBackend(BaseMicrositeTemplateBackend):
     """
     Specialized class to pull templates from the database
     """
+    def get_template_path(self, relative_path, **kwargs):
+        return relative_path
```

Comment:
Right, gotcha. But I'm wondering about mixing-and-matching, say the config coming from the database but the templates from disk. Let me see if we can get this get_template_path into the Template backends.